proxy/src/logging.rs

```diff
@@ -28,10 +29,10 @@ pub fn init() {
                 };
                 writeln!(
                    fmt,
-                    "{} {} {:?}{}",
+                    "{} {}{} {}",
                     level,
-                    record.target(),
                     Context(&ctxt.borrow()),
+                    record.target(),
```

Comment:
I wonder if there's a way to include the context information in the target of the log message instead, so that we can filter certain contexts in the log configuration string (the way we can for modules), without breaking the ability to filter by modules as well? Just a thought, and not something we have to do in _this_ PR...

Reply:
I agree that this sort of feature would be nice, but it's a drastically different feature (this starts to become a tracing problem, really). Let's solve this problem separately.

Reply:
Yeah, definitely.